tests/functional/watcher_controller_test.go

```diff
@@ -52,6 +61,13 @@ var _ = Describe("Watcher controller with minimal spec values", func() {
 			}, timeout, interval).Should(ContainElement("openstack.org/watcher"))
 		})
 
+		It("It has the expected container image defaults", func() {
+			Watcher := GetWatcher(watcherTest.Instance)
+			Expect(Watcher.Spec.APIContainerImageURL).To(Equal(util.GetEnvVar("RELATED_IMAGE_WATCHER_API_IMAGE_URL_DEFAULT", watcherv1beta1.WatcherAPIContainerImage)))
```

Comment:
are those `*_URL_DEFAULT` vars defined in these tests? I think we should simply use the `watcherv1beta1` variables here

Reply:
Done!